pkg/bff/credentials.go

```diff
@@ -0,0 +1,150 @@
+package bff
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	jwt "github.com/golang-jwt/jwt/v4"
+	"github.com/trisacrypto/directory/pkg/bff/config"
+	"github.com/trisacrypto/directory/pkg/gds/admin/v2"
+	"github.com/trisacrypto/directory/pkg/gds/tokens"
+)
+
+// Credentials implements the admin.Credentials interface to provide access and refresh
+// tokens to the GDSClient.
+type Credentials struct {
+	tm      *tokens.TokenManager
+	access  string
+	refresh string
```

Comment:
Given that the credentials have the token manager, I don't think a refresh token is necessary. Refresh tokens reduce the chance that you have to resubmit your username/password if you're already authenticated. But they just hand out a new access token which this struct could just as easily generate itself.